ospool-pilot/itb/pilot/where-am-i

```diff
@@ -101,20 +104,20 @@ else
   gconfig_add "PublicIPV6" "Unknown"
 fi
 
-ping -c 1 google.com
-ping_error_code=$?
+python3 ospool-pilot/itb/lib/tcpping.py -n 1 google.com
```

Comment:
I don't know if this path is always going to be correct, I suggest doing something similar to how `ospool-lib` is referenced in other scripts:

https://github.com/opensciencegrid/osg-flock/blob/d84736f344f96073b91ef8a57300d307944684cf/ospool-pilot/itb/pilot/default-image#L173-L181

You could then point a variable at the path and use that variable whenever you want to run `tcpping.py`:

```shell
group_dir=$(get_glidein_config_value GLIDECLIENT_GROUP_WORK_DIR)
client_dir=$(get_glidein_config_value GLIDECLIENT_WORK_DIR)
cp -ns $client_dir/tcpping.py $group_dir/ 2>/dev/null && echo "Linking tcpping.py from $(ls $client_dir/tcpping.py | tr '\n' ' ')" 1>&2

TCPPING="python $group_dir/tcpping.py"

TCPPING -n 1 google.com
tcpping_error_code=$?
...
tcpping_output="$(${TCPPING} -n 15 osg-kansas-city...)"
```
etc.